### PR TITLE
feat(Webhook): add 'fetchMessage' method

### PR DESCRIPTION
**Pull request by izexi**
Ref: https://github.com/discord/discord-api-docs/pull/2812
~~This PR will be marked as draft until the referenced PR above has been merged.~~

**Status and versioning classification:**
- Code changes have been tested against the Discord API, or there are no code changes
- I know how to update typings and have done so, or typings don't need updating
- This PR changes the library's interface (methods or parameters added)

### CODE REVIEW

**Comment by ThaTiemsz** [comment]
has been merged now in https://github.com/discord/discord-api-docs/commit/94d57ea3ed3b2c42cbbb488c9f93d2bc81706cff. On a side note, shouldn't the jsdocs/typings for the method also accept a string since `@original` is also accepted as parameter?

File: src/structures/Webhook.js [hunk 1]
```diff
@@ -226,6 +226,18 @@ class Webhook {
     return this;
   }
 
+  /**
+   * Gets a message that was sent by this webhook.
+   * @param {Snowflake} message The ID of the message to fetch
```
Comment (ckohen) [inline src/structures/Webhook.js:231]:
```suggestion
   * @param {MessageResolvable} message The ID of the message to fetch
```

Mirror this from the similar methods (and possibly add `@original` to the typedef for MessageResolvable, that I'm not sure on though).

**Comment by izexi** [comment]
> has been merged now in [discord/discord-api-docs@94d57ea](https://github.com/discord/discord-api-docs/commit/94d57ea3ed3b2c42cbbb488c9f93d2bc81706cff). On a side note, shouldn't the jsdocs/typings for the method also accept a string since `@original` is also accepted as parameter?

Although `Snowflake` is currently typed as a `string` and passing `@original` would work, I do agree that it would be nicer to have a union type that also accepts `@original` as a string literal.

I was going to comment on this to be added in #5448 after this PR has been merged, since I think its more within the scope of that PR (and the same will need to be done on `Webhook#editMessage/deleteMessage()`).

File: src/structures/Webhook.js (hunk 1 above)
Comment (izexi) [reply src/structures/Webhook.js:231]: I did consider using a `MessageResolvable` type at first but [`MessageManager#fetch()`](https://github.com/izexi/discord.js/blob/feat/Webhook%23fetchMessage()/typings/index.d.ts#L2043) accepts a `Snowflake` as the parameter so I went ahead and used that here as well to be consistent with that. Which similar methods are you referring to here?

Comment (ckohen) [reply src/structures/Webhook.js:231]: I misread actually, but I was referencing editMessage and deleteMessage. You are directly passing the message token here so MessageResolvable is actually incorrect 🤦

**Review by catcosmicice — APPROVED** [review]

**Review by kyranet — APPROVED** [review]

**Review by iCrawl — APPROVED** [review]

**Review by SpaceEEC — APPROVED** [review]

**Pull request merged**